src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests.cs

```diff
@@ -5012,11 +5193,22 @@ public C(object b)
 }
 ";
             var compilation = CreateCompilationWithMscorlib45(source, options: TestOptions.DebugExe);
+#if ALLOW_IN_CONSTRUCTOR_INITIALIZER
```

Comment:
The purpose is to keep expectations, we are planning to allow declarations in these contexts in the future.